source/includes/connection-snippets/scram-snippets/nodejs-connection.js

```diff
@@ -0,0 +1,10 @@
+// begin scram connection
+const MongoClient = require('mongodb').MongoClient;
+const uri = "mongodb+srv://<username>:<password>@<your-cluster-url>/test?retryWrites=true&w=majority";
+const client = new MongoClient(uri, { useNewUrlParser: true });
+client.connect(err => {
+  const collection = client.db("test").collection("devices");
```

Comment:
The other SCRAM driver snippets only retrieve a reference to the database. Could you make sure that they are consistent in which reference they store?

Reply:
The scram snippets are actually verabitim copied and pasted from their driver docs. But I can fix them to be more consistent